### PR TITLE
BaSP-TG-126-130: Update README and Update Sidebar

### PR DESCRIPTION
**Pull request by laura-brussa12**
WIP

**TG 130: Update README:**

- Update the list of the members 

**TG 126: Update Sidebar:**

- Create a share component named "NavLink"
- Using that to show the links depending the user (admin, super-admin or employee)

### CODE REVIEW

File: src/components/Shared/NavLink/index.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          <li className={styles.items}>
+        <NavLink to={'/projects'} className={styles.links}>
+        Projects
+        </NavLink>
+        </li>
+        <li className={styles.items}>
+        <NavLink to={'/time-sheets'} className={styles.links}>
+        Timesheets
+        </NavLink>
+      </li> */}
```
Comment (HernanCazeneuve) [inline src/components/Shared/NavLink/index.js:115]: don't know what must be done with the commented lines

**Review by HernanCazeneuve — COMMENTED** [review]
Great job, see comment

**Review by GuidoCerioni — APPROVED** [review]
🚀

**Review by tobiaspujol — APPROVED** [review]
Good job!

**Pull request merged**